constrain/library/fan_static_pressure_reset_control.py

```diff
@@ -1,67 +1,31 @@
 from constrain.checklib import RuleCheckBase
-from datetime import date
 
 
 class FanStaticPressureResetControl(RuleCheckBase):
     points = [
         "p_set",
-        "p_set_min",
         "d_VAV_1",
         "d_VAV_2",
         "d_VAV_3",
         "d_VAV_4",
         "d_VAV_5",
-        "tol",
     ]
 
     def verify(self):
         d_vav_points = ["d_VAV_1", "d_VAV_2", "d_VAV_3", "d_VAV_4", "d_VAV_5"]
         d_vav_df = self.df[d_vav_points]
-        self.df["result"] = True
 
         for row_num, (index, row) in enumerate(self.df.iterrows()):
             if row_num != 0:
                 if (d_vav_df.loc[index] > 0.9).any():
-                    self.df.at[index, "result"] = True
+                    self.df.at[index, "result"] = "Untested"
```

Comment:
I don't think that this is correct. We want to check that the setpoint is reset until one of the the box is nearly wide open, so if we see a box that's almost fully open we probably want to return `True` as in the control strategy is correct. Actually, I think that this could be injected in the `else` statement below: 1) we check that the `p_set` of the previous timestep is lower than the current, 2) if that's not the case, is one of the boxes fully opened? If so, the test passes, otherwise it fails.

Reply:
Good point. Addressed.